logstash-core/build.gradle

```diff
@@ -121,8 +121,7 @@ dependencies {
     compile "com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:${jacksonVersion}"
     compile "org.jruby:jruby-complete:${jrubyVersion}"
     compile 'com.google.googlejavaformat:google-java-format:1.5'
-    testCompile 'org.apache.logging.log4j:log4j-core:2.6.2:tests'
-    testCompile 'org.apache.logging.log4j:log4j-api:2.6.2:tests'
+    testCompile 'org.apache.logging.log4j:log4j-core:2.9.1:tests'
```

Comment:
I don't think this is even needed ... 
Per https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_plugin_and_dependency_management

"Additional dependencies for compiling tests"

Since it is already a compile dependency, i don't think we need it as testCompile dependency

Reply:
@jakelandis careful this one has the `tests` classifier, we need the log4j test libs for the test logging context that we use in our logging tests => we need it 4 sure :)

Reply:
ah... missed that, thanks for the clarification.